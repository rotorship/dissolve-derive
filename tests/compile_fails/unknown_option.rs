use dissolve_derive::Dissolve;

#[derive(Dissolve)]
struct UnknownOption {
	#[dissolved(unknown)]
	field: String,
}

fn main() {}
