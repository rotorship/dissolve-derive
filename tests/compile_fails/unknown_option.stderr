error: unknown dissolved attribute option 'skip'; supported options: rename, unknown = "new_name"
 --> tests/compile_fails/unknown_option.rs:5:14
  |
5 |     #[dissolved(unknown)]
  |                 ^^^^^^^
